README.md

```diff
@@ -264,6 +264,7 @@ Name | Website | Region
 [FingerprintJS](/company-profiles/fingerprintjs.md) | https://fingerprintjs.com | Worldwide
 [Fire Engine Red](/company-profiles/fire-engine-red.md) | https://fire-engine-red.com/ | Worldwide
 [Fireball Labs](/company-profiles/fireball-labs.md) | https://www.fireballlabs.com | Germany
+[FIS GLOBAL](/company-profiles/FIS-global.md) | https://www.fisglobal.com/ | USA, UK, Asia
```

Comment:
Me the file reference lowercase here @ashishdawale20

Reply:
My bad, corrected it now.